### PR TITLE
Feat router pages and authentication

### PR DESCRIPTION
**Pull request by ankit9015**
This PR has following feature:
- React router is added to navigate pages
- Authentication pages : Login & Signup
- Authentication and private routes

https://user-images.githubusercontent.com/67504902/166151647-7d981b2d-02f1-4779-8509-cc8e8ceda2d1.mp4

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
 import "./App.css";
+import "./constants/utility.css";
 import { Footer, Header, Main, Sidebar } from "./pageLayout";
+import { useLocation } from "react-router-dom";
 
 function App() {
   const [showSidebar, setShowSidebar] = useState(false);
+  const forbiddenPages = ["/login", "/signup", "*"];
+  const currentLocation = useLocation().pathname;
+  console.log(currentLocation);
```
Comment (rushikesh3299) [inline src/App.js:11]: you can remove console logs before pushing changes

**Review by rushikesh3299 — APPROVED** [review]
Looks good to me

File: src/App.js (hunk 1 above)
Comment (ankit9015) [reply src/App.js:11]: Thanks, I missed it

**Review by JantuDeb — APPROVED** [review]
This really good I didn't find any issues with the code.
I would suggest you to divide your code with reusable component.

**Pull request merged**